scripts/visualize/visualize-doy.R

```diff
@@ -38,10 +41,13 @@ visualize.doy <- function(viz = getContentInfo(viz.id = "doy-NM")){
   #################
   xml_name(doy_svg, ns = character()) <- "g"
   xml_attr(doy_svg, "class") <- "axis-labels"
```

Comment:
axis lines end up in this group ('axis-labels' class). Do you want different grouping for the lines, or a different class name?